File: examples/hello.py
#!/usr/bin/env -S uv --quiet run --script
# /// script
# requires-python = ">=3.12"
# dependencies = [
# "ffmpeg-normalize",
# ]
# ///

def day_kind(day):
    match day:
        case ("Monday"
            | "Tuesday"
            | "Wednesday"
            | "Thursday"
            | "Friday"):
            return "Weekday"
        case "Saturday" | "Sunday":
            return "Weekend"

print(f"Hello it is a {day_kind('Tuesday')}")

File: examples/hello.ts
#!/usr/bin/env bun
import { $ } from 'bun';
// --- Import packages ---
await $`bun add --global lodash || true`.quiet()

// --- Script starts here ---
import _ from 'lodash'; // External packages

const out = _.chain(["Hello", "World"]).join(" ").value()

console.log(out + "!")
